source/blog/2021-11-09-announcing-hanami-200alpha3.html.markdown

```diff
@@ -0,0 +1,163 @@
+---
+title: Announcing Hanami v2.0.0.alpha3
+date: 2021-11-09 12:30:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Streamlined source directories, more flexible settings, better integrated actions, and the beginning of a monthly release cadence.
+---
+
+Hello Hanami community! It’s Tim here again, and I’m delighted to announce the release of Hanami 2.0.0.alpha3!
+
+This release represents months of work on several foundational aspects of the framework, and brings:
+
+- Streamlined source directories for easier-to-navigate project file trees
```

Comment:
How about this?

```suggestion
- Shallow application directories
```

Reply:
Sure, if you prefer that, happy to change it